doc/source/start.rst

```diff
@@ -3,85 +3,125 @@ Getting Started
 
 Installation
 ------------
-Bandit is distributed on PyPI. The best way to install it is with pip:
 
+Bandit is distributed on PyPI. The best way to install it is with pip.
 
-Create a virtual environment (optional)::
+Create a virtual environment (optional):
+
+.. code-block:: console
 
     virtualenv bandit-env
     python3 -m venv bandit-env
-    # And activate it:
+
+And activate it:
+
+.. code-block:: console
+
     source bandit-env/bin/activate
 
-Install Bandit::
+Install Bandit:
+
+.. code-block:: console
 
     pip install bandit
 
-Run Bandit::
+If you want to include TOML support, install it with the `toml` extras:
+
+.. code-block:: console
+
+    pip install bandit[toml]
```

Comment:
I just noticed that `tomli` is required if you use a `pyproject.toml`. In my case our CI didn't break because it was installed via other packages, but a bare install makes it indeed not work.

Since the `setup.cfg` file seems to also have an extra for `yaml` files:

https://github.com/PyCQA/bandit/blob/caae4eef975e49a8fea10a12327b088ce12e9981/setup.cfg#L30-L34

Should we mention it as well in here? Or does a `.bandit` YAML file work with just installing `pip install bandit`?

Reply:
Hmm, I'm confused. :thinking:  Installing the `toml` extra, that's what the `pip install bandit[toml]` does. Anything else needed from your point of view in addition to those instructions?

Reply:
What I mean is that in order to make bandit process a `pyproject.toml` file it needs to be installed via `pip install bandit[toml]`. Does it need to be installed via `pip install bandit[yaml]` for yaml files? If so, it should probably be in this docs as well.

Reply:
I understand! So, "If you want to include TOML support" is not clear enough?

Reply:
Argh, YAML! Now I understand. Let me verify this locally.

Reply:
I can confirm, the latest version of PyYAML (*not* the one in [requirements.txt](https://github.com/PyCQA/bandit/blob/main/requirements.txt#L5)) is installed as a dependency when you install Bandit via `pip install bandit`.

I'm a bit puzzled: I absolutely can't see in the source code where the dependencies being installed are defined.